### ISSUE TITLE
[FEATURE] Access to logits and final hidden layer 

### ISSUE THREAD

**Issue by lshamis** [microsoft/DeepSpeed-MII#463]
Hello, I don't believe there is any way to access logits or the final hidden layer embedding in the current API and would like to request this feature.
Thank you

**Comment by SulRash** [comment]
This would definitely make the framework infinitely better than vLLM, which suffers the same issue.
